src/main/java/org/gridsuite/study/server/repository/StudyEntity.java

```diff
@@ -172,6 +173,10 @@ public class StudyEntity extends AbstractManuallyAssignedIdentifierEntity<UUID>
     @Builder.Default
     private StudyIndexationStatus indexationStatus = StudyIndexationStatus.NOT_INDEXED;
 
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "study_voltage_init_parameters_id")
```

Comment:
Set explicit and readable name for foreign key constraint

Reply:
I thought that was quite explicit already, it's parameters related to both study and voltage init parameters. Do you have any other idea ?

Reply:
It's for the foreign key name
See securityAnalysisParameters in the same entity just above with JPA annotation : 
foreignKey = @ForeignKey(
                    name = "securityAnalysisParameters_id_fk"
            ))

Reply:
ok fixed !